tdp/core/runner/deployment_plan.py

```diff
@@ -51,6 +52,14 @@ def from_dag(
     def from_operations(operations):
         deployment_args = dict(
             targets=[operation.name for operation in operations],
-            using_dag=False,
+            deployment_type=DeploymentTypeEnum.OPERATIONS,
+        )
+        return DeploymentPlan(operations, deployment_args)
+
+    @staticmethod
```

Comment:
This should take a DeploymentLog as input (not operations), and should fail if the deploymentlog is in success